src/main/java/com/bandwidth/sdk/messaging/models/MessageEvent.java

```diff
@@ -18,6 +20,17 @@
     public abstract Optional<Integer> getErrorCode();
     public abstract Optional<String> getTo();
 
+
+    @JsonIgnore
+    public boolean isIncomingMessage(){
+        return getTo().isPresent();
+    }
+
+    @JsonIgnore
+    public boolean isDLR(){
```

Comment:
I think we should try not using acronyms here. Most people outside of the industry probably don't know what it is.
`isDeliveryReceipt()`?